src/Tests/Ingest/Processor/IngestProcessorGrokUnitTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Linq;
+using FluentAssertions;
+using Nest;
+using Tests.Framework;
+
+namespace Tests.XPack.IngestProcessorGrok
+{
+	public class IngestProcessorGrokUnitTests
+	{
+		[U]
+		public void ShouldDeserialize()
```

Comment:
Not sure this test is needed; Can `IngestProcessorGrokApiTests ` assert that the response contains at least one pattern?

Reply:
Hmmm. I had thought about that, but I didnt want to make any assumptions...